lib/jekyll/theme.rb

```diff
@@ -38,12 +37,6 @@ def assets_path
       @assets_path ||= path_for "assets".freeze
     end
 
-    def configure_sass
```

Comment:
I can't remember - why did we remove this again? jekyll-sass-converter now handles themes itself?

Reply:
> why did we remove this again?

If you look at the method's implementation, it calls the `Sass` module (from erstwhile gem `sass`) creating an unhealthy dependency on a library managed by the sass-converter plugin.
By removing the method Jekyll Core has completely yielded all control to the plugin. The converter in turn does everything it's meant to do only interfacing with Jekyll Core via **expected and documented API**s (e.g. the Converter API endpoint `#convert(content)`; the Generator API endpoint `#generate(site)`, etc.


> jekyll-sass-converter now handles themes itself?

Yes. It does via public method `Jekyll::Converters::Scss#sass_load_paths`
```ruby
def sass_load_paths
  paths = user_sass_load_paths + [sass_dir_relative_to_site_source]
  paths << site.theme.sass_path if site.theme&.sass_path
  ...
end
```